tests/python/pants_test/engine/legacy/test_build_ignore_integration.py

```diff
@@ -73,6 +82,7 @@ def test_build_ignore_dependency(self):
                                 })
 
     self.assert_failure(run_result)
+
```

Comment:
Should probably revert this for smaller diff, unless you meant to add it for a specific reason.